app/main.py

```diff
@@ -1 +1,51 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
```

Comment:
Try to do one attribute per line. so it will looks like
```
    def __init__(
         self,
         name: str,
         appetite: int,
         is_hungry: bool = True
    ) -> None:
```
And `) -> None` always below of method/function name started